bin/test-browser.js

```diff
@@ -181,90 +136,62 @@ function BenchmarkReporter(runner) {
   });
 }
 
-function startTest() {
+async function startTest() {
 
-  console.log('Starting', client, 'on', testUrl);
+  console.log('Starting', browserEngine, 'on', testUrl);
 
-  var opts = {
-    browserName: client.browser,
-    version: client.version,
-    platform: client.platform,
-    tunnelTimeout: testTimeout,
-    name: client.browser + ' - ' + tunnelId,
-    'max-duration': 60 * 45,
-    'command-timeout': 599,
-    'idle-timeout': 599,
-    'tunnel-identifier': tunnelId
-  };
-  if (CHROME_BIN) {
-    opts.chromeOptions = {
-      binary: CHROME_BIN,
-      args: ['--headless', '--disable-gpu', '--no-sandbox', '--disable-setuid-sandbox']
-    };
-  }
-  if (FIREFOX_BIN) {
-    opts.firefox_binary = FIREFOX_BIN;
-  }
+  const browserImpl = playwrightBrowsers[browserEngine];
 
-  var runner = new RemoteRunner();
+  const runner = new RemoteRunner();
   new MochaSpecReporter(runner);
   new BenchmarkReporter(runner);
 
-  seleniumClient.init(opts, function (err) {
-    if (err) {
-      testError(err);
-      return;
-    }
-    console.log('Initialized');
+  const options = {
+    headless: true,
+  };
+  const browser = await browserImpl.launch(options); // FIXME Or 'firefox' or 'webkit'.
```

Comment:
is the fixme still relevant? it seems you select the browser engine from the import not via options...

Reply:
Good catch :+1: